src/workerd/api/sql.c++

```diff
@@ -62,6 +62,45 @@ bool SqlStorage::allowTransactions() const {
   return false;
 }
 
+jsg::JsValue SqlStorage::wrapSqlValue(jsg::Lock& js, SqlValue value) {
+  KJ_IF_SOME(v, value) {
+    KJ_SWITCH_ONEOF(v) {
+      KJ_CASE_ONEOF(bytes, kj::Array<byte>) {
+        return jsg::JsValue(js.wrapBytes(kj::mv(bytes)));
+      }
+      KJ_CASE_ONEOF(text, kj::StringPtr) {
+        return js.str(text);
+      }
+      KJ_CASE_ONEOF(number, double) {
+        return js.num(number);
+      }
+    }
+    KJ_UNREACHABLE;
+  } else {
+    return js.null();
+  }
+}
+
+jsg::JsObject SqlStorage::wrapSqlRow(jsg::Lock& js, SqlRow row) {
+  return jsg::JsObject(js.withinHandleScope([&]() -> v8::Local<v8::Object> {
+    jsg::JsObject result = js.obj();
+    for (auto& field: row.fields) {
+      result.set(js, field.name, wrapSqlValue(js, kj::mv(field.value)));
+    }
+    return result;
+  }));
+}
+
+jsg::JsArray SqlStorage::wrapSqlRowRaw(jsg::Lock& js, kj::Array<SqlValue> row) {
+  return jsg::JsArray(js.withinHandleScope([&]() -> v8::Local<v8::Array> {
+    v8::LocalVector<v8::Value> values(js.v8Isolate);
```

Comment:
```suggestion
    v8::LocalVector<v8::Value> values(js.v8Isolate);
    values.reserve(row.size());
```

Reply:
Good optimization but I have to merge this tonight and I need to sign off shortly so as soon as CI is green I'm going to merge, will do this in some future PR.